haskell/actions.bzl

```diff
@@ -455,8 +465,7 @@ def create_ghc_package(ctx, interfaces_dir, static_library, dynamic_library):
     "id": get_pkg_id(ctx),
     "key": get_pkg_id(ctx),
     "exposed": "True",
-    "exposed-modules":
-      " ".join([module_name(ctx, f) for f in _hs_srcs(ctx)]),
+    "exposed-modules": " ".join(public_modules),
```

Comment:
Why are we not listing hidden modules in `hidden-modules` field? I think we should have it.

Reply:
Good catch, adding.